libraries/AP_ADSB/AP_ADSB.h

```diff
@@ -209,6 +233,9 @@ class AP_ADSB {
         bool        is_flying;
         bool        is_in_auto_mode;
 
+        bool        ident_pending;
+        bool        ident_isActive;
```

Comment:
"ident_isActive" should probably be renamed to "ident_is_active" to fit with our use of small case and underscores instead of camel case.

Reply:
renamed and added comment and context